extensions/github/src/unread-notifications.tsx

```diff
@@ -33,6 +33,18 @@ function UnreadNotifications() {
     return response.data;
   });
 
+  const hasUnread = data && data.length > 0;
+
+  async function markAllNotificationsAsRead() {
+    try {
+      showHUD("Marking All as Read");
+      await mutate(octokit.rest.activity.markNotificationsAsRead());
+      showHUD("All have been marked as Read");
```

Comment:
Let's remove the first `showHUD` since this will lead to closing Raycast's main window
```suggestion
      await mutate(octokit.rest.activity.markNotificationsAsRead());
      showHUD("Marked all notifications as read");
```